### DISCUSSION TITLE
Load .hic file for large genome & chromosomes

### DISCUSSION THREAD

**Discussion by tbrown91** [GMOD/jbrowse-components#2780] [General] [answered]
Hi,

I am trying to load a .hic file into jbrowse2 v1.6.5 desktop and am getting nothing showing up. This genome is 20Gb in size and the smallest chromosome is about 700Mb. I was able load genome and gff (with .csi indexing) fairly ok, but the hic track is just blank.

I first started with the whole-genome hic file (17Gb) and then one for only the smallest chromosome (224Mb) but both show nothing. It's difficult for me to say anything more as there is no error message given.

Here is the example juicebox image for smallest chromosome:

![image](https://user-images.githubusercontent.com/11500978/156178014-1af75797-beb8-4274-aba3-01f1dd1135e6.png)

Is there a way to check .hic file formatting is correct, or to potentially see what the issue may be?

Thanks,

Tom

**Comment by cmdcolin** [comment, marked as answer]
thanks for the feedback :) definitely curious...are you able to load the data in juicebox.js (https://aidenlab.org/juicebox/)? we use the same library for parsing the .hic files as them. the other thing that can sometimes cause tracks to not show up is "refname aliasing" e.g. using chr1 vs 1, in which case the refNameAliases can be configured, or the files can be standardized on the same naming scheme

**Reply by tbrown91** [reply]
Thanks for getting back to me so quickly! It seems to have worked ok. I only loaded the single chromosome, which is why you can barely see it in the whole picture view:

![image](https://user-images.githubusercontent.com/11500978/156188983-df6014fd-57d5-4cb5-b3c6-f1d2a54d0f69.png)

![image](https://user-images.githubusercontent.com/11500978/156189074-8b9b2e24-d1d6-455a-9647-4bf31c630a82.png)

The only immediate difference I can see is that the text is capitalised in the juicebox view - is that an issue?

**Reply by tbrown91** [reply]
I'm not sure if it helps, but this is the file info inside jbrowse:

![image](https://user-images.githubusercontent.com/11500978/156189649-17f34152-e207-4370-b810-af2571eb1140.png)

It looks like at least the header is correct as it knows the resolution. I used the chromosome sizes file instead of a genome name - again, could that be an issue?

**Reply by cmdcolin** [reply]
the upper case vs lower case chromosome names could be a factor....if you want to send me the file (chrom sizes and hic) i can check it out also

**Reply by tbrown91** [reply]
Do you need the genome as well, or are the sizes and hic file enough?

Do you have an upload link in case everything is too big?

**Reply by cmdcolin** [reply]
chrom.sizes and hic should be enough (as long as the chrom.sizes names match the fasta to make sure though)

I don't have an upload link myself but google drive or something like that might work

**Reply by tbrown91** [reply]
You were right! I changed the chromosome names in the fast to capitals and it loaded. It's a bit of a pain that juicer_tools pre did this, but at least I have a workaround now. 

Thanks for your help.

**Reply by cmdcolin** [reply]
excellent...jbrowse 2 is pretty strict about names matching while other tools might smooth this over. glad this worked out though :) let us know if anything else comes up
